src/appleseed/renderer/kernel/rendering/localsampleaccumulationbuffer.cpp

```diff
@@ -259,6 +259,9 @@ void LocalSampleAccumulationBuffer::develop_to_frame(
     // Copy tile first so the critical section can end before doing the expensive develop_to_tile.
     const AccumulatorTile& level = *m_levels[m_active_level];
     AccumulatorTile& read_level = *m_read_levels[m_active_level];
+    const std::uint32_t active_level = m_active_level.value(); // Ensure the same active_level is used for both arrays.
+    const AccumulatorTile& level = *m_levels[active_level];
+    AccumulatorTile& read_level = *m_read_levels[active_level];
```

Comment:
You declare level and read_level two times. That won't build, see the last build log: https://travis-ci.org/github/appleseedhq/appleseed/jobs/738901308#L1880

Isn't this better ? 

```suggestion
    const std::uint32_t active_level = m_active_level.value(); // Ensure the same active_level is used for both arrays.
    const AccumulatorTile& level = *m_levels[active_level];
    AccumulatorTile& read_level = *m_read_levels[active_level];    
```

Reply:
Sorry about that. This is what I intended to do. Something must have gone wrong when merging the fix from somewhere else. Thanks for the fix.